### ISSUE TITLE
[Bug]: Revives are instant (no ace)

### ISSUE THREAD

**Issue by SilenceIsFatto** [igorkis-scrts/A3-Antistasi-Plus#184]
### Describe the bug

If you have the medic trait, you will instantly revive anyone you attempt to revive

(This bug report came from "drunkonsuccess" on discord, I fixed it for Ultimate)

### How to reproduce

Revive some AI without ace loaded, whilst being a combat life saver

### Version

Current

### Have you altered the code?

Yes

### What i have changed

```markdown
Old:

private _timer = if (_isMedic) then { A3A_reviveTime / 2 } else { time + A3A_reviveTime };

New:

private _timer = if (_isMedic) then { time + (A3A_reviveTime / 2) } else { time + A3A_reviveTime };

Before it was just "A3A_reviveTime / 2" which I believe was an oversight?
```

### Map

N/A

### What server?

LAN Hosted

### Time bug occured (Server time/UTC)

N/A

### Mods

```html
N/A
```

### Additional context

I would have created a PR but I already have a fork (Ultimate) so it's being weird

File path: A3A\addons\core\functions\Revive\fn_actionRevive.sqf

To recap (Line 40-50):

Before:
`private _timer = if (_isMedic) then { A3A_reviveTime / 2 } else { time + A3A_reviveTime };`

After:
`private _timer = if (_isMedic) then { time + (A3A_reviveTime / 2) } else { time + A3A_reviveTime };`

**Comment by igorkis-scrts** [comment]
Thanks for ticket, yeah, I got reports that something wrong with revives since the last update, guess this is it.

**Comment by igorkis-scrts** [comment]
fixed in 2.3
